plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -701,6 +706,28 @@ public void testShowTables()
         assertFileSystemAccesses("SHOW TABLES", ImmutableMultiset.of());
     }
 
+    @Test
+    public void testReadMultipartCheckpoint()
+            throws Exception
+    {
+        String tableName = "test_multipart_checkpoint_" + randomNameSuffix();
+        Path tableLocation = Files.createTempFile(tableName, null);
+        copyDirectoryContents(new File(Resources.getResource("deltalake/multipart_checkpoint").toURI()).toPath(), tableLocation);
+
+        assertUpdate("CALL system.register_table('%s', '%s', '%s')".formatted(getSession().getSchema().orElseThrow(), tableName, tableLocation.toUri()));
+        assertFileSystemAccesses("SELECT * FROM " + tableName,
+                ImmutableMultiset.<FileOperation>builder()
+                        .addCopies(new FileOperation(LAST_CHECKPOINT, "_last_checkpoint", INPUT_FILE_NEW_STREAM), 1)
```

Comment:
use `add` for entries with 1 copy
the `addCopies` should stand out, since every such is a potential improvement.